test/integration/run-test.sh

```diff
@@ -62,22 +62,22 @@ fi
 
 # Begin to run CSI functions one by one
 echo 'Create volume test:'
-readonly value=$("$CSC_BIN" controller new --endpoint "$endpoint" --cap 1,block "$volname" --req-bytes 2147483648 --params "$params")
```

Comment:
I am not sure why these integration use block access_type. @andyzhangx I am changing this to mount. But please feel free to raise any concern if there is any.

Reply:
thanks, that's correct fix.